README.rst

```diff
@@ -1,5 +1,5 @@
 
-.. image:: https://user-images.githubusercontent.com/14353512/199113556-4b53660f-c628-4138-8d01-3719595ecda1.png
+.. image:: https://raw.githubusercontent.com/OpenEnergyPlatform/open-MaStR/feature-490-add-offis-as-partner/docs/images/README_HeaderThreePartners.svg
```

Comment:
As soon as the branch is deleted, this image won't be available no more, will it?

Reply:
Do you know how to get this user-images link? For the original one I imagine it went this way:
* Having a README in .md format
* Drag and drop the image in the github editor
* Convert .md to .rst

Now that we have .rst we cannot drag and drop the image to the README anymore.

One solution is to copy the image to the issue #490 and use the generated link from there, hoping that issue images will not be deleted sometimes.

Reply:
Yes this looks like a very good solution 👍

Reply:
I hope it will also work for the pypi page of open-mastr

Reply:
> I hope it will also work for the pypi page of open-mastr

The image is located at a separate location (pypi-camo.freetls.fastly.net...) so probably copied on release. Not sure but I guess it will work out..